TerminalDocs/tutorials/new-tab-same-directory.md

```diff
@@ -90,11 +93,11 @@ If you're using [Starship](http://starship.rs/), then that will already modify y
 
 ```powershell
 function Invoke-Starship-PreCommand {
-  $loc = $($executionContext.SessionState.Path.CurrentLocation);
+  $loc = $executionContext.SessionState.Path.CurrentLocation;
   $prompt = "$([char]27)]9;12$([char]7)"
```

Comment:
what does 9;12 do anyway?

Reply:
From [ConEmu specific OSC](https://conemu.github.io/en/AnsiEscapeCodes.html#ConEmu_specific_OSC)

Sequence | Description
-- | --
ESC ] 9 ; 12 ST | Let ConEmu treat current cursor position as prompt start. Useful with PS1.

Looks like it's about the same as a `OSC 133 ; A`. Looks like we don't support it at the moment but that would be _trivial_ to fix